WordPress/Docs/PHP/WordPress.WP.AlternativeFunctions.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="WordPress Alternative Functions"
+    >
+    <standard>
+    <![CDATA[
+    Use WordPress functions instead of native PHP functions to maintain compatibility and benefit from WordPress's additional security and performance improvements.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Using wp_safe_redirect().">
+        <![CDATA[
+        wp_safe_redirect( $url );
+        ]]>
+        </code>
+        <code title="Invalid: Using PHP's header() function for redirection.">
+        <![CDATA[
+        header( "Location: $url" );
```

Comment:
Correct me if I'm missing something, but as far as I can test, the use of `header()` does not trigger the `WordPress.WP.AlternativeFunctions`. I created a file with the contents of this invalid example and run PHPCS but got no errors:

```
$ cat test.inc 
<?php

header( "Location: $url" );
$ vendor/bin/phpcs --standard=WordPress test.inc -s --sniffs=WordPress.WP.AlternativeFunctions
$
```

I haven't tested all the other `<code_comparison>` blocks, but on a quick look, it seems to me `file_get_contents()` does this this sniff, but the others don't. Now I'm starting to question my self if I'm checking the correct sniff. This is the one that I'm looking: https://github.com/WordPress/WordPress-Coding-Standards/blob/38c950e7c45326b6f49796145a0cb3e62d981b1d/WordPress/Sniffs/WP/AlternativeFunctionsSniff.php#L27. Before I continue this review, could you please check and get back to me?